README.md

```diff
@@ -7,17 +7,18 @@ Yet another F-Droid client.
 
 ## Description
 
-Unofficial F-Droid client that resembles the classic F-Droid client.
+Unofficial F-Droid client in the style of the classic one.
 
-The client supports all basic F-Droid features, such as repository management and package installation. It doesn't
-support privileged extension, root installation, or local repositories.
+Jump over the lazy dog, manage repositories, and install software quickly.
+No privileged extension, root installation, or sharing local repositories nearby.
 
 ### Features
 
-* Classic F-Droid style, without cards or inappropriate animations
-* Fast repositories synchronization under good network conditions
-* Built upon standard Android components and a minimal set of dependencies
-* Made with attention to details, both in program logic and design
+* Classic F-Droid style.
+* No cards or inappropriate animations.
+* Fast repository syncing.
+* Standard Android components and minimal dependencies.
+* Attention to detail in programming logic and design.
```

Comment:
"programming logic" concerns me here. Initially it was "program logic" like "logic in program code", not like "logic in programming". I think "program logic" is more correct.

Reply:
Actually, now I think that it would be better to remove this line at all.